src/main/java/org/jitsi/videobridge/cc/SimulcastController.java

```diff
@@ -288,7 +299,7 @@ public boolean accept(RawPacket pkt)
         if (currentBaseLayerIndex != targetBaseLayerIndex)
         {
             // We do want to switch to another layer
-            if (shouldSwitchToStream(currentIndex, sourceLayerIndex, targetIndex))
+            if (shouldSwitchToStream(currentIndex, sourceLayerIndex, targetIndex, sourceEncodings))
```

Comment:
This is why the original code was comparing TL0 indices. Can we change this fix to match the original behavior?